apps/api/src/app/events/usecases/send-message/send-message.usecase.ts

```diff
@@ -116,12 +116,7 @@ export class SendMessage {
   private async filter(command: SendMessageCommand) {
     const data = await this.getFilterData(command);
 
-    const configuration = {
-      job: command.job,
-      command,
-    };
-
-    const shouldRun = await this.matchMessage.filter(command.step, data, configuration);
```

Comment:
we actually don't need to create this `configuration` object here (which has command in it), but instead, we can pass the `command`